pykusto/_src/functions.py

```diff
@@ -996,14 +996,14 @@ def to_datetime(expr: StringType) -> _DatetimeExpression:
         return _DatetimeExpression(KQL(f'todatetime({_to_kql(expr)})'))
 
     @staticmethod
-    def to_decimal(expr: NumberType) -> _NumberExpression:
+    def to_decimal(expr: ExpressionType) -> _NumberExpression:
```

Comment:
The `PythonTypes` are supported, since these methods accept literal arguments, for example: `todecimal("123")`.
The argument can also be a general expression, it doesn't have to be as single column, for example: `todecimal(strcat("12", "34"))`